src/Microsoft.ML.TensorFlow/TensorFlow/TensorGeneric.cs

```diff
@@ -67,6 +68,10 @@ public static TFTensor CreateScalar<T>(T data)
             {
                 return new TFTensor((System.UInt16)(object)data);
             }
+            else if (typeof(T) == typeof(System.ReadOnlyMemory<char>))
+            {
+                return CreateString(Encoding.UTF8.GetBytes(((System.ReadOnlyMemory<char>)(object)data).ToArray()));
```

Comment:
>(System.ReadOnlyMemory<char>)(object) [](start = 60, length = 37)

Is this double casting necessary?

Reply:
Yes, generic type needs casting this way.

---
In reply to: [257013294](https://github.com/dotnet/machinelearning/pull/2546#discussion_r257013294) [](ancestors = 257013294)